<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>213</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Label_3">
         <property name="toolTip">
          <string>Number of events analyzed</string>
         </property>
         <property name="text">
          <string>Events processed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Label_5">
         <property name="toolTip">
          <string>Analysis error rate</string>
         </property>
         <property name="text">
          <string>Processing error:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Label">
         <property name="toolTip">
          <string>Molecule partitioning rate</string>
         </property>
         <property name="text">
          <string>Capture Rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="Label_7">
         <property name="toolTip">
          <string>Percent completed</string>
         </property>
         <property name="text">
          <string>Progress:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Label_9">
         <property name="toolTip">
          <string>Elapsed time</string>
         </property>
         <property name="text">
          <string>Elapsed Time:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Label_11">
         <property name="toolTip">
          <string>Analysis time remaining</string>
         </property>
         <property name="text">
          <string>Remaining Time:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Label_10">
         <property name="toolTip">
          <string>Processing time per event</string>
         </property>
         <property name="text">
          <string>Time Per Event:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="neventsLabel">
         <property name="text">
          <string>n/a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorrateLabel">
         <property name="text">
          <string>n/a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="caprateLabel">
         <property name="text">
          <string>n/a</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QProgressBar" name="analysisprogressBar">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="progressLabel">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>n/a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="walltimeLabel">
         <property name="text">
          <string>n/a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remaintimeLabel">
         <property name="text">
          <string>n/a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timepereventLabel">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>n/a</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
